bless/request/bless_request.py

```diff
@@ -9,6 +9,7 @@
 
 # man 8 useradd
 USERNAME_PATTERN = re.compile('[a-z_][a-z0-9_-]*[$]?\Z')
+HOSTNAME_PATTERN = re.compile('[a-z0-9_.-]+')
```

Comment:
Do you have hostnames with an _ in them?
https://en.wikipedia.org/wiki/Hostname#Restrictions_on_valid_host_names